cobbler.spec

```diff
@@ -291,7 +292,7 @@ if [ "$1" = "upgrade" ]; then
 if [ $1 -ge 2 ]; then
 %endif
     # package upgrade: backup configuration
-    DATE=$(date "+%Y%m%d-%H%M%S")
+    DATE=$(date "+%%Y%%m%%d-%%H%%M%%S")
```

Comment:
I'm not sure what this improves over the current `DATE`.

Reply:
Exactly - it fails to build with the latest version of rpm.